_apps/boring-cyborg.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Boring Cyborg
+description: Automatically label PRs based on the path of the file that are modified in the PR
+slug: boring-cyborg
+screenshots:
+- https://github.com/kaxil/boring-cyborg/blob/master/assets/usage-screenshot-1.png
+authors: [ "kaxil" ]
+repository: kaxil/boring-cyborg
+host: https://gentle-mesa-48041.herokuapp.com/
```

Comment:
This looks like a temporary URL. We have a check in place that pings this URL regularly to make sure the app is still working. Do you plan to change it anytime soon?

Reply:
No, the app is deployed on Heroku. Currently it is used by Apache Airflow project. 

I will definitely update the url in this markdown when I change the url.

Reply:
Perfect, thanks!